pixi.toml

```diff
@@ -455,9 +472,8 @@ pip = ">=23"
 platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
 
 [feature.python-pypi.pypi-dependencies]
-rerun-sdk = "==0.16.1"
-# TODO(jprochazk): Uncomment this once we have published the notebook package to pypi
-#rerun-notebook = "==0.16.1"
+rerun-sdk = "==0.17.0"
+rerun-notebook = "==0.17.0"
```

Comment:
Ideally we update the `release.yml` workflow to also update that version. Let's at least add a TODO to that effect.

Reply:
too late 😅